src/site/content/en/blog/content-indexing-api/index.md

```diff
@@ -0,0 +1,363 @@
+application.
+
+1. Make sure that you're using a supported browser and platform—currently,
+   that's limited to **Chrome 80 or later on Android**.
+1. Visit [https://contentindex.dev](https://contentindex.dev)
+1. Click on the `+` button next to one or more of the items on the list.
+1. [Optional] Disable your device's WiFi and cellular data connection, or enable
+   airplane mode to simulate taking your browser offline.
+1. Choose "Downloads" from Chrome's menu, and switch to the "Articles for You" tab.
+1. Browse through the content that you previously saved.
+
+You can view the source of the sample application [on
+GitHub](https://github.com/rayankans/contentindex.dev).
+
+Another sample application, a [Scrapbook PWA](https://scrapbook-pwa.web.app/),
+illustrates the use of the Content Indexing API in conjunction with the [Web
+Share Target API](/web-share-target/). The [code demonstrates a
+technique](https://github.com/GoogleChrome/samples/blob/gh-pages/web-share/src/js/contentIndexing.js)
+for keeping the Content Indexing API in sync with items stored by a web app
+using the Cache Storage API.
+
+## Using the API {: #using-the-api }
+
+To use the API your app must have a service worker and URLs that are navigable
+offline. If your web app does not currently have a service worker, the [Workbox
+libraries](https://developers.google.com/web/tools/workbox/) can simplify
+creating one.
+
+### What type of URLs can be indexed as offline-capable? {: #offline-capable-urls }
+
+The API supports indexing URLs corresponding to HTML documents. A URL for cached
+media file, for example, can't be indexed directly. Instead, you need to provide
+a URL for a page that displays media, and which works offline.
+
+A recommended pattern is to create a "viewer" HTML page that could accept the
+underlying media URL as a query parameter and then display the contents of the
+file, potentially with additional controls or content on the page.
+
+Web apps can only add URLs to the content index that are under the
+[scope](https://developers.google.com/web/ilt/pwa/introduction-to-service-worker)
+of the current service worker. In other words, a web app could not add a URL
+belonging to a completely different domain into the content index.
+
+### Overview {: #api-overview }
+
+The Content Indexing API supports three operations: adding, listing, and
+removing metadata. These methods are exposed from a new property, `index`, that
+has been added to the
+<code>[ServiceWorkerRegistration](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerRegistration)</code>
+interface.
+
+The first step in using any of the Content Indexing API methods is therefore to
+get a reference to the current `ServiceWorkerRegistration`. Within a web page,
+the most straightforward way to do that is to use the
+<code>[navigator.serviceWorker.ready](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerContainer/ready)</code>
+promise:
```

Comment:
This paragraph could be simplified. Maybe: 

'The first step in indexing content is getting a reference to the current `ServiceWorkerRegistration`. Using `navigator.serviceWorker.ready` is the most straightforward way.

'using any of the Content Indexing API methods' I didn't use this construction because readers understand from the context what API you're talking about.

Reply:
I've rewritten it along the lines of your suggestion.